### PR TITLE
chore(dapp): Re-frame tests w/karma for local + CI testing

### PR DESCRIPTION
**Pull request by drew-patel**
# Description

- Updates some namespace refs which were failing test compilation
- Adds re-frame/test + shadow-cljs aliases for testing events and subscriptions
- Basic test for wallet init
- More tests to be written once wallet functions are ported to `:browser` target 

Updates 2/4 tasks in https://github.com/kubelt/kubelt/issues/317

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [x] Ran tests that were written in this PR locally 
- [x] Updated GH workflows to run them on PR + merge

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

#New tests passing on CI

Screen Shot 2022-04-20 at 16 10 21

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,6 +60,7 @@
     "karma": "6.3.17",
     "karma-chrome-launcher": "3.1.1",
     "karma-cljs-test": "0.1.0",
+    "karma-junit-reporter": "^2.0.1",
```
Comment (crimeminister) [inline package.json:63]:
```suggestion
    "karma-junit-reporter": "2.0.1",
```

File: shadow-cljs.edn [hunk 2]
```diff
@@ -30,10 +30,12 @@
   [com.github.mainej/headlessui-reagent "1.5.0.47"]
   ;; immutable database and Datalog query engine
   [datascript/datascript "1.3.12"]
-  ;; tracing for re-frame events
-  [day8.re-frame/tracing "0.6.2"]
   ;; inspection tool for re-frame events & app-db
-  [day8.re-frame/re-frame-10x "1.2.5"]
+  ^:dev [day8.re-frame/re-frame-10x "1.2.5"]
```
Comment (crimeminister) [inline shadow-cljs.edn:34]: Using `^:dev` for this purpose is new to me; neat!

**Review by crimeminister — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (drew-patel) [reply package.json:63]: Will drop. Thanks!

Comment (drew-patel) [reply package.json]: Done

**Review by szkl — APPROVED** [review]

**Review by tangrammer — APPROVED** [review]
Looks great!

**Review by crimeminister — APPROVED** [review]

**Pull request merged**